src/Filter/Sprockets.php

```diff
@@ -139,21 +139,23 @@ public function getDependencies(AssetTarget $target)
         $children = [];
         foreach ($target->files() as $file) {
             $contents = $file->contents();
-            preg_match($this->_pattern, $contents, $matches);
+            $count = preg_match_all($this->_pattern, $contents, $matches);
             if (empty($matches)) {
                 continue;
             }
 
-            if ($matches[1] === '"') {
-                // Same directory include
-                $path = $this->_findFile($matches[2], dirname($file->path()) . DIRECTORY_SEPARATOR);
-            } else {
-                // scan all paths
-                $path = $this->_findFile($matches[2]);
+            for ($i = 0; $i < $count; $i ++) {
+                if ($matches[1][$i] === '"') {
+                    // Same directory include
+                    $path = $this->_findFile($matches[2][$i], dirname($file->path()) . DIRECTORY_SEPARATOR);
+                } else {
+                    // scan all paths
+                    $path = $this->_findFile($matches[2][$i]);
+                }
+                $dep = new Local($path);
+                $children[] = $dep;
```

Comment:
Let's suppose we have the following lines in `application.js`.
```js
//= require <jquery.min.js>
//= require <underscore-min.js>
//= require "base/base.js"
```
I found that application.js does not update when underscore-min.js or base.js files are modified. But it did update when jquery.min.js file was changed.
`preg_match` in the original code returns only the first match. We need global matching, so I thought `preg_match` should be replaced with `preg_match_all` and had to write some code to handle arrays accordingly.